### DISCUSSION TITLE
Doc re: Changing package access seems out-dated

### DISCUSSION THREAD

**Discussion by RavenHursT** [npm/feedback#710] [Documentation] [answered]
The page here says that logging in as an admin, paid account, I should A) see an "admin" button or link and B) Should see a dropdown or selector of some sort to change the access status of the package.

I am logged in as a paid admin account for the package I want to change.  But I see no "admin" link/tab/button, rather there's a "settings" tab.  Under "settings", I can see the access status as "public", but there's nothing in the UI to _change_ that status.

<img width="776" alt="image" src="https://user-images.githubusercontent.com/496983/174135996-beeab69e-1aea-49ca-b07a-d76211431725.png">

Is the _only_ way to do this through the CLI?

Update:

Apparently not 😢 

```
npm access restricted @my-org/my-package
npm ERR! code E503
npm ERR! 503 Service Unavailable - POST https://registry.npmjs.org/-/package/%40my-org%2Fmy-package/access - Service Unavailable
```

**Comment by monishcm** [comment, marked as answer]
I believe you are paid user, but the scenario you are trying is for a paid org. To take a package under an org. private, you need to pay for the org.

[Upgrading to a paid organization plan](https://docs.npmjs.com/upgrading-to-a-paid-organization-plan)

Meanwhile we've taken feedback on the documentation and is in our backlog now.

**Reply by RavenHursT** [reply]
@monishcm My user is indeed the owner of the organization that the package I'm trying to convert is under.

<img width="901" alt="Screen Shot 2022-06-17 at 3 24 04 PM" src="https://user-images.githubusercontent.com/496983/174408712-c913c7d9-952c-4a8d-912b-99ef185cfac8.png">


And there is _no_ "admin" link/tab and no way to change the access status of the package.

**Comment by RavenHursT** [comment]
Oh.. mah bad... yup.. while I'm a paid user.. Now that I'm looking at the billing for the _org_, turns out, no one's paid for the org yet 🤦‍♂️

Sorry for the 🧠🌬️
